db/models.py

```diff
@@ -40,8 +48,95 @@ def __str__(self) -> str:
 
 class MovieSession(models.Model):
     show_time = models.DateTimeField()
-    cinema_hall = models.ForeignKey(to=CinemaHall, on_delete=models.CASCADE)
-    movie = models.ForeignKey(to=Movie, on_delete=models.CASCADE)
+    cinema_hall = models.ForeignKey(
+        to="CinemaHall", on_delete=models.CASCADE, related_name="cinema_halls"
```

Comment:
Here's the same issue.
By the way, why set `related_name` only to some of the fields? It seems like an inconsistency that will make your models confusing to use.